### PR TITLE
FIX parent_id param name

### PR DESCRIPTION
**Pull request by jlejeune**
Now, save to dir works

### CODE REVIEW

File: putio.py [hunk 1]
```diff
@@ -321,7 +321,7 @@ def get(cls, id):
     @classmethod
     def add_url(cls, url, parent_id=0, extract=False, callback_url=None):
         d = cls.client.request('/transfers/add', method='POST', data=dict(
-            url=url, save_parent_id=parent_id, extract=extract,
+            url=url, parent_id=parent_id, extract=extract,
```
Comment (cenk) [inline putio.py]: ```/transfers/add``` accepts ```save_parent_id```, not ```parent_id```. We should not change it.

See docs: https://api.put.io/v2/docs/transfers.html#add

File: putio.py [hunk 2]
```diff
@@ -331,7 +331,7 @@ def add_torrent(cls, path, parent_id=0, extract=False, callback_url=None):
         with open(path, 'rb') as f:
             files = {'file': f}
             d = cls.client.request('/files/upload', method='POST', files=files,
-                                   data=dict(save_parent_id=parent_id,
+                                   data=dict(parent_id=parent_id,
```
Comment (cenk) [inline putio.py]: This is okay.

File: putio.py (hunk 1 above)
Comment (jlejeune) [inline putio.py]: My bad, a little too quick to fix :p

**Comment by jlejeune** [comment]
Just updated to modify add_torrent function only

**Comment by cenk** [comment]
Released new version to PyPI: https://pypi.python.org/pypi/putio.py/3.1.1

**Pull request merged**